content/docs/command-reference/remote/modify.md

```diff
@@ -134,6 +134,14 @@ these settings, you could use the following options.
   $ dvc remote modify myremote --local credentialpath /path/to/my/creds
   ```
 
+- `config` - S3 config file path. Supports some s3-specific
+  [configuration options](https://docs.aws.amazon.com/cli/latest/topic/s3-config.html#aws-cli-s3-configuration)
```

Comment:
is there a default value for this? do we read by default at all? It would be great to specify the behavior a bit.